prov/verbs/src/ep_rdm/verbs_av_ep_rdm.c

```diff
@@ -157,6 +161,23 @@ static int fi_ibv_rdm_av_insert(struct fid_av *av_fid, const void *addr,
 			FI_INFO(&fi_ibv_prov, FI_LOG_AV,
 				"fi_av_insert: bad addr #%i\n", i);
 
+			if(av->eq && (av->flags & FI_EVENT))
+			{
+				/* due to limited functionality of
+				 * verbs EQ notify last failed element
+				 * only. */
+				/* TODO: what about utils EQ? */
+				struct fi_eq_err_entry err = {
+					.fid = &av->av_fid.fid,
+					.context = context,
+					.data = i,
+					.err = FI_EINVAL,
+					.prov_errno = FI_EINVAL
+				};
+				av->eq->err = err;
```

Comment:
The compiler will likely drop the variable anyway, but why not just initialize av->eq->err directly?  If you can use the util EQ, that would be best, though as a separate patch is fine,